### ISSUE TITLE
sidebar behave weird when immediately click on fade area right after click on Learn tab

### ISSUE THREAD

**Issue by 66eli77** [learningequality/ka-lite#3843]
Browser: Chrome
Mac installer
Reproduce: 1. click on learn tab; 2 click on the fade area within 0.1 second after click on learn tab.(it's important to click it very fast)
![screen shot 2015-06-12 at 4 08 42 pm](https://cloud.githubusercontent.com/assets/5577205/8141571/cb6c3d16-111d-11e5-97a6-f4b7a4ccd1ed.png)

**Comment by rtibbles** [comment]
This is probably a  result of our reliance on debounce for some of the callbacks, 100ms is the threshold being used, I believe!

**Comment by benjaoming** [comment]
I have never once in my time with KA Lite provoked this. @radinamatic have you seen it occur?

**Comment by radinamatic** [comment]
Yes, remember seeing that occur intermittently in the past, but not recently.
Kudos to @66eli77 👍  for nailing it in the screenshot, I never managed to grab it! 😂
